modules/microshift-storage-about-vol-snapshots.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// microshift/volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="about-volume-snapshots_{context}"]
+= About volume snapshots
+
+Volume snapshots can be used with LVM thin volumes to help protect against data loss from applications running in a {product-title} cluster. {product-title} only supports the LVMS CSI provider.
```

Comment:
Abbreviations need to be spelled out the first time they are used in a module.

Reply:
Consider wording to what the user can do with it

```suggestion
You can use volume snapshots with LVM thin volumes to help protect against data loss from applications running in a {product-title} cluster. {product-title} only supports the LVMS CSI provider.
```